Client/Frontend/UserContent/UserScripts/BraveSearchHelper.js

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is missing the Copyright portion of the header, see https://github.com/brave/brave-core/blob/114c50ede570d53f7add02d212481914b30ab7ff/chromium_src/android_webview/renderer/aw_key_systems.cc#L1-L4 for an example.

I know that there are lots of files which are missing this in the iOS codebase, but we should make sure it's present in new files we add.

Reply:
Added followup ticket https://github.com/brave/brave-ios/issues/3782